### ISSUE TITLE
embeddings

### ISSUE THREAD

**Issue by dmarx** [manovaboard/Suicide_Watch#3]
Incorporate pre-trained word embeddings to convert word tokens into a lower-dim representation for the network. Easiest options:

* google's word2vec
* glove

[Demo](https://github.com/A-Jacobson/CNN_Sentence_Classification/blob/master/WordVectors.ipynb) via [this pytorch issue](https://discuss.pytorch.org/t/can-we-use-pre-trained-word-embeddings-for-weight-initialization-in-nn-embedding/1222/18)
